sites/skeleton.dev/src/content/docs/components/toast/react.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import Example from '@examples/components/toast/Example.tsx';
+import ExampleRaw from '@examples/components/toast/Example.tsx?raw';
+import ExampleType from '@examples/components/toast/ExampleType.tsx';
+import ExampleTypeRaw from '@examples/components/toast/ExampleType.tsx?raw';
+import ExamplePlacement from '@examples/components/toast/ExamplePlacement.tsx';
+import ExamplePlacementRaw from '@examples/components/toast/ExamplePlacement.tsx?raw';
+import ExamplePromise from '@examples/components/toast/ExamplePromise.tsx';
+import ExamplePromiseRaw from '@examples/components/toast/ExamplePromise.tsx?raw';
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Example client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Type
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<ExampleType client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ExampleTypeRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+Note: You can also use the more generic `create` method in combination with the `type` property when needing to create dynamic toast notifications.
+
+## Placement
+
```

Comment:
We shouldn't shy away from being explicit with what's being shown. We defaulted to titles early on to keep things simple, but I'd recommend a brief explain nation like:

> Use `placement` to control the display position on the screen. This setting is shared between all toast instances.